framework/tester.py

```diff
@@ -144,11 +144,22 @@ def start_all_servers(self):
             if not srv.is_running():
                 raise Exception("Can not start server %s" % id)
 
+    def wait_all_connections(self, tmt=1):
+        for id in self.__servers:
+            srv = self.__servers[id]
+            self.assertTrue(srv.wait_for_connections(timeout=tmt))
```

Comment:
I think it's not a good idea to put assert here

Reply:
Also, for not started servers, wait_for_connections() return False.

Reply:
Corrected.